torch/fx/experimental/proxy_tensor.py

```diff
@@ -419,7 +419,7 @@ def extract_val(val: _ExtractValType) -> _ExtractValType:
     elif val is None:
         return None
 
-    typing_extensions.assert_never(val)
+    typing_extensions.assert_never(val)  # type: ignore[arg-type]
```

Comment:
This type ignore will disable `assert_never`. You should keep `py_sym_types` unannotated and let `mypy` infer the type.

Reply:
@pytorchbot merge -r